code/game/machinery/planet_beacon.dm

```diff
@@ -0,0 +1,216 @@
+#define BEACON_UNANCHORED 1
```

Comment:
Remember to `#undef` your defines at the end of the file if they are not used anywhere else.